apps/metadata-editor-e2e/src/e2e/edit.cy.ts

```diff
@@ -1,12 +1,49 @@
+/* eslint-disable cypress/no-unnecessary-waiting */
+
 describe('editor form', () => {
-  beforeEach(() => {
+  let recordUuid
+  before(() => {
     cy.login('admin', 'admin', false)
 
-    // Alpine convention record
-    cy.visit('/edit/8698bf0b-fceb-4f0f-989b-111e7c4af0a4')
-
     cy.clearRecordDrafts()
+    // Duplicate a record to manipulate it at will
+    cy.visit('/catalog/search')
+    cy.get('.table-header-cell').eq(1).click()
+    cy.get('.table-header-cell').eq(1).click()
+    cy.get('[data-cy="table-row"]')
+      .first()
+      .find('[data-test="record-menu-button"]')
+      .click()
+    cy.get('[data-test="record-menu-duplicate-button"]').click()
+    cy.wait(1200)
+    cy.get('md-editor-publish-button').click()
+
+    cy.visit('/catalog/search')
+    cy.get('.table-header-cell').eq(1).click()
+    cy.get('.table-header-cell').eq(1).click()
+    cy.get('[data-cy="table-row"]').first().children('div').eq(2).click()
+
+    cy.get('gn-ui-record-form')
+      .children('div')
+      .first()
+      .children('div')
+      .eq(1)
+      .find('gn-ui-form-field')
+      .eq(0)
+      .find('gn-ui-form-field-simple')
+      .find('input')
```

Comment:
9 operators to find the correct element... Couldn't we had some data-cy attributes as shortcut selector?

Reply:
True, I kind of forgot about this one